### PR TITLE
Working on importing student list 

### PR DESCRIPTION
**Pull request by RobertBasile-1**
Having this issue when trying to upload student from excel  

image

### CODE REVIEW

**Review by Akram-El-Gaouny — APPROVED** [review]

File: server/main.py [hunk 1]
```diff
@@ -1,8 +1,24 @@
+from flask import Flask, request, jsonify
+import openpyxl
 from flask_restful import Api
 from routes.student import StudentAPI, StudentsAPI
+import pandas as pd
 
+app = Flask(__name__)
```
Comment (Laith-Grira) [inline server/main.py:7]: This is already called in the main.py

File: server/main.py [hunk 2]
```diff
@@ -1,8 +1,24 @@
...
+@app.route("/import-excel", methods=["POST"])
+def import_excel():
+    file = request.files["file"]
+    if not file:
+        return "No file selected", 400
+    data = pd.read_excel(file,na_values=["N/A", "na", "--","NaN", " "])
+    return data.to_json(orient="records")
+
+if __name__ == "__main__":
+    app.run()
```
Comment (Laith-Grira) [inline server/main.py:24]: is it possible that we add those in the route? (the run() app is also called in run.py)
Otherwise we can leave that for another PR if you want, once we start testing the coode

File: server/main.py (hunk 1 above)
Comment (Laith-Grira) [reply server/main.py:7]: Or probably you can move the code to main.py for now, because Flask() is called there

**Review by Laith-Grira — APPROVED** [review]
Thanks Rob! 🐐

**Review by Akram-El-Gaouny — APPROVED** [review]

**Review by Laith-Grira — APPROVED** [review]
